### PR TITLE
util-linux: 2.36.1 -> 2.36.2

### PR DESCRIPTION
**Pull request by ajs124**
###### Motivation for this change
Upstream release.

Includes #97952 (cc @uosis)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by drewrisinger — CHANGES_REQUESTED** [review]
* [ ] Diff: Might need license update
* [x] Commits LGTM
* [ ] Didn't attempt build

File: pkgs/os-specific/linux/util-linux/default.nix [hunk 1]
```diff
@@ -71,6 +66,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://www.kernel.org/pub/linux/utils/util-linux/";
     description = "A set of system utilities for Linux";
+    changelog = "https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v${lib.versions.majorMinor version}/v${version}-ReleaseNotes";
     license = licenses.gpl2; # also contains parts under more permissive licenses
```
[the remark this answers is not in the document]
Comment (ajs124) [reply pkgs/os-specific/linux/util-linux/default.nix:70]: https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/tree/README.licensing

So this should probably be something like `with licenses; [ gpl2Only gpl2Plus gpl3Plus lgpl21Plus bsd3 publicDomain ];` and I don't think we have anything for "BSD 4-Clause University of California-Specific", so idk what to do about that?

Should I add that?

[the remark this answers is not in the document]
Comment (drewrisinger) [reply pkgs/os-specific/linux/util-linux/default.nix:70]: I suppose so, but I'd have to defer to someone else. I don't have any experience with multi-licenses for Nix derivations.

**Comment by drewrisinger** [comment]
FWIW, I built ``cmake`` with this PR, and it built fine. Didn't try anything else, though.

**Comment by alyssais** [comment]
@OfBorg test boot

**Pull request merged**